From 0eb0938878d65353ed5e77285616fbda1bc34abc Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:14:50 -0400
Subject: [alsa] oxygen: add mute mask for the OXYGEN_PLAY_ROUTING register

Message-id: <140972823573-49-patchset-jkysela@redhat.com>
Patchwork-id: 90749
O-Subject: [RHEL 7.1 PATCH 048/442] ALSA: oxygen: add mute mask for the OXYGEN_PLAY_ROUTING register
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

The Xonar DG/DGX driver needs this mask to mute unnecessary
channels.

Signed-off-by: Roman Volkov <v1ron@mail.ru>
Signed-off-by: Clemens Ladisch <clemens@ladisch.de>

(picked from kernel: 079e0cb776cf52b09fae89551b8a7eaff5639166)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/oxygen/oxygen_regs.h b/sound/pci/oxygen/oxygen_regs.h
index 63dc7a0..8c191ba 100644
--- a/sound/pci/oxygen/oxygen_regs.h
+++ b/sound/pci/oxygen/oxygen_regs.h
@@ -318,6 +318,7 @@
 #define  OXYGEN_PLAY_MUTE23  0x0002
 #define  OXYGEN_PLAY_MUTE45  0x0004
 #define  OXYGEN_PLAY_MUTE67  0x0008
+#define  OXYGEN_PLAY_MUTE_MASK  0x000f
 #define  OXYGEN_PLAY_MULTICH_MASK 0x0010
 #define  OXYGEN_PLAY_MULTICH_I2S_DAC 0x0000
 #define  OXYGEN_PLAY_MULTICH_AC97 0x0010
-- 
1.7.1